packages/core/CalendarManager.ts

```diff
@@ -199,20 +200,24 @@ const getMonths = (dateFrom: string, dateTo: string): string[] => {
 };
 
 export const getBusyCalendarTimes = async (
```

Comment:
did you make sure to update every instance of this function?

Reply:
yeah this would've been caught out by type checks